src/range_methods.jl

```diff
@@ -0,0 +1,145 @@
+## BOUNDEDNESS TRAIT
+
+# For random search and perhaps elsewhere, we need a variation on the
+# built-in boundedness notions:
+abstract type PositiveUnbounded <: Unbounded end
```

Comment:
Why a d difference between PositiveUnbounded and RightUnbounded ?

Reply:
Well, initially I thought of translating distributions when fitting to a right unbounded range with `lower` not equal to zero. But the sparsity of use-cases I could imagine did not seem to justify the much larger extra implementation complexity. So now, if you are positive unbounded range `r`, then the value of `r.lower` is used only to truncate, not to fit. To fit we just look at `unit` and `origin` to fit the (two-parameter) distribution specified (and then truncate). Most of the time `r.lower=0` and you are probably specifying a positive distribution type to fit anyway. 

At the end of day, if you don't like the automatically fit distribution, you can specify your own.